src/Sarif.Multitool.Library/Rules/RuleResources.Designer.cs

```diff
@@ -978,5 +1068,32 @@ internal class RuleResources {
                 return ResourceManager.GetString("SARIF2016_FileUrisShouldBeRelative_Note_ShouldNotStartWithSlash_Text", resourceCulture);
             }
         }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Azure DevOps Advanced Security.
+        /// </summary>
```

Comment:
Why is there only one period at the end of the sentence here and two in the user-facing strings above? If there's no good reason, should be consistently one period throughout.

Reply:
This is a string that is inserted into other strings. The period in this case is added by VS for this comment. When you see strings with two period, the first is in the resource string itself (end of sentence).